templates/barbican/config/barbican-api-config.json

```diff
@@ -78,14 +78,16 @@
       "optional": true,
       "merge": true
     },
+    {{- if (index . "P11ClientDataPath") }}
```

Comment:
if we default `P11ClientDataPath` to some value if `ClientDataPath` is not set, then we could just add that entry with out the if, we already set `"optional": true,` so its fine to not have files

Reply:
Done